lib/ansible/modules/cloud/vmware/vmware_portgroup.py

```diff
@@ -20,70 +21,78 @@
 version_added: 2.0
 author:
 - Joseph Callen (@jcpowermac)
 - Russell Teague (@mtnbikenc)
-- Abhijeet Kasurde (@Akasurde) <akasurde@redhat.com>
+- Abhijeet Kasurde (@Akasurde)
+- Christian Kotte (@ckotte)
 notes:
-    - Tested on vSphere 5.5, 6.5
+    - Tested on vSphere 5.5 and 6.5
+    - Complete configuration only tested on vSphere 6.5
 requirements:
     - "python >= 2.6"
     - PyVmomi
 options:
-    switch_name:
+    switch:
         description:
             - vSwitch to modify.
         required: True
-    portgroup_name:
+        aliases: [ 'switch_name', 'vswitch' ]
+    portgroup:
         description:
             - Portgroup name to add.
         required: True
+        aliases: [ 'portgroup_name' ]
     vlan_id:
         description:
             - VLAN ID to assign to portgroup.
-        required: True
-    network_policy:
+            - Set to 0 (no VLAN tagging) by default.
+        required: False
+        default: 0
+        aliases: [ 'vlan' ]
+    security:
         description:
             - Network policy specifies layer 2 security settings for a
               portgroup such as promiscuous mode, where guest adapter listens
               to all the packets, MAC address changes and forged transmits.
             - Dict which configures the different security values for portgroup.
             - 'Valid attributes are:'
-            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: false)'
-            - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: false)'
-            - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: false)'
+            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: None)'
+            - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: None)'
+            - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: None)'
         required: False
-        version_added: "2.2"
-        default: {
-            mac_changes: false,
-            promiscuous_mode: false,
-            forged_transmits: false,
-        }
-    teaming_policy:
+        version_added: "2.8"
+        aliases: [ 'security_policy' ]
+    teaming:
         description:
             - Dictionary which configures the different teaming values for portgroup.
             - 'Valid attributes are:'
-            - '- C(load_balance_policy) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
+            - '- C(load_balancing) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
             - '   - choices: [ loadbalance_ip, loadbalance_srcmac, loadbalance_srcid, failover_explicit ]'
-            - '- C(inbound_policy) (bool): Indicate whether or not the teaming policy is applied to inbound frames as well. (default: False)'
```

Comment:
Any specific reason for remove `inbound_policy` and `rolling_order` ?

Reply:
`inbound_policy` is a deprecated option according to the API documentation and it's also not available in the GUI.

`rolling_order` is called `failback` in the GUI and the behavior is also the opposite: `rolling_order = False` is actually `failback = True`. We should name the options as they appear in the GUI and not in the API. The user doesn't know the API.

However, I could add them again plus some code and a deprecation note that it will be removed in 2.11.

Reply:
> `inbound_policy` is a deprecated option according to the API documentation and it's also not available in the GUI.
> 
> `rolling_order` is called `failback` in the GUI and the behavior is also the opposite: `rolling_order = False` is actually `failback = True`. We should name the options as they appear in the GUI and not in the API. The user doesn't know the API.
> 

OK
> However, I could add them again plus some code and a deprecation note that it will be removed in 2.11.

That would be great. Let me know once you have that change in place.